Backend/Services/EnemyService.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.AspNetCore.Cors.Infrastructure;
+using Domain.Entities;
+using System;
+using Backend.Repositories;
+
+
+namespace Backend.Services
+{
+    public class EnemyService : IEnemyService
+    {
+
+        private readonly IEnemyRepository _enemyRepository;
+
+        public EnemyService(IEnemyRepository enemyRepository)
+        {
+            _enemyRepository = enemyRepository;
+        }
+
+        public Enemy CreateEnemy(Enemy enemy)
+        {
+            _enemyRepository.AddEnemy(enemy);
+
+            return enemy;
+        }
+
+        public void DeleteEnemy(int id)
```

Comment:
Vi skal sørge for at der ikke kan gå ting galt undervejs når vi kører programmet. Da vi sad og kiggede på det sammen, så fejlede det hvis man slettede den samme 2 gange. 
plz implementer at den ikke kommer til at fejle ved det